src/like-service/config/app.php

```diff
@@ -52,7 +52,7 @@
     |
     */
 
-    'url' => env('APP_URL', 'http://localhost'),
+    'url' => env('APP_URL', 'http://0.0.0.0:8000/like-service'),
```

Comment:
[DISCUSS] I think we should use the environment variables here to build the app url

Reply:
enviroment variables are now used